selenium_tests/ab_tool/page_objects/ab_add_experiment_page.py

```diff
@@ -0,0 +1,63 @@
+"""This is the page object of the Add Experiment Page"""
+from selenium.webdriver.common.by import By
+from selenium.webdriver.common.keys import Keys
+from selenium_tests.ab_tool.page_objects.ab_base import ABBasePage
+from selenium.common.exceptions import NoSuchElementException
+
+
+### Locators for the Add_experiment page###
+class Locator2(object):
+    _experimentname = (By.ID, "experimentName") 
+    _notes_field = (By.ID, "experimentNotes")
+    _new_track_name = (By.ID,"newTrackName")
+    _add_tracks = (By.ID, "addTrack") 
+    _submit_button = (By.ID, "create-confirm")
+    _confirm_create = (By.ID, "createNow")
+    _uniform_checkbox = (By.ID, "uniformRandom")
+    _trash_icon = (By.CSS_SELECTOR, "i.fa.fa-trash")
+
+class AddExperimentPage(ABBasePage):
+    def __init__(self, driver):
+        super(AddExperimentPage, self).__init__(driver)  
+    
+
+#These are actions that can be taken on the page as part of adding a track
+
+    def create_name (self,driver,my_experiment_name):
```

Comment:
This is just filling in the experiment name, right?  Maybe call it `set_experiment_name()` instead?